src/models/generated/HourlyMatching.jl

```diff
@@ -0,0 +1,63 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
```

Comment:
Also, what is this about?

Reply:
Yeah we definitely need a proper docstring here